.circleci/config.yml

```diff
@@ -30,14 +30,15 @@ jobs:
       - run:
           name: Check maven pom files format (lint)
           command: |
+            uname -a
```

Comment:
Nothing, I put it when debugging and forgot to remove it, but I think is not problem, in any case, just a useful message in case of changes for the `image: maven:3.6-jdk-11` image